airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -60,6 +60,7 @@ def __init__(self,
             gcp_conn_id=bigquery_conn_id, delegate_to=delegate_to)
         self.use_legacy_sql = use_legacy_sql
         self.location = location
+        self.num_retries = self._get_field('num_retries', 5)
```

Comment:
I really like that you take values from parameters of Google Connection :)

Reply:
If anyone is interested in this subject then the source of the idea is this comment: https://github.com/apache/airflow/pull/5054#discussion_r272843031